content/documentation/pages/3-stream-developer-guides/2-streams/1-standalone-stream-rabbitmq.md

```diff
@@ -0,0 +1,207 @@
+
+##### Initialzr Shortcut
+
+Another option instead of using the UI to initialize your project you can do the following:
+
+1. Click the [here](https://start.spring.io/starter.zip?fakeusernameremembered=&fakepasswordremembered=&type=maven-project&language=java&bootVersion=2.1.4.RELEASE&baseDir=usage-cost-processor&groupId=io.spring.dataflow.sample&artifactId=usage-cost-processor&name=usage-cost-processor&description=Sample+project+for+Spring+Cloud+Stream+Processor&packageName=com.example.demosource&packaging=jar&javaVersion=1.8&inputSearch=&style=amqp&style=cloud-stream) to download the preconfigured usage-cost-processor.zip.
+2. Unzip the usage-cost-processor.zip file and import the project into your favorite IDE
+
+#### Biz Logic
+
+1.  In your favorite IDE create the `io.spring.dataflow.sample.domain` package
+1.  Create a `UsageDetail` class in the `io.spring.dataflow.sample.domain` using your favorite IDE that looks like the contents in [UsageDetail.java](https://github.com/spring-cloud/spring-cloud-dataflow-samples/blob/master/dataflow-website/stream-developer-guides/streams/usage-detail-sender/src/main/java/com/example/demo/domain/UsageDetail.java).
+    This `UsageDetail` model contains `userId`, `data` and `duration`.
+1.  Create a `UsageCostDetail` class in the `io.spring.dataflow.sample.domain` using using your favorite IDE that looks like the contents in [UsageCostDetail.java](https://github.com/spring-cloud/spring-cloud-dataflow-samples/blob/master/dataflow-website/stream-developer-guides/streams/usage-cost-processor/src/main/java/com/example/demo/domain/UsageCostDetail.java).
+    This `UsageCostDetail` model contains `userId`, `callCost` and `dataCost`.
+1.  Create the `Processor` application that receives the `UsageDetail` from the previously created `source`, computes the call/data cost and returning the `UsageCostDetail`.
+
+    ```java
+    @EnableBinding(Processor.class)
+    public class UsageCostProcessor {
+
+    private double ratePerSecond = 0.1;
+
+    private double ratePerMB = 0.05;
+
+    @StreamListener(Processor.INPUT)
+    @SendTo(Processor.OUTPUT)
+    public UsageCostDetail processUsageCost(UsageDetail usageDetail) {
+    UsageCostDetail usageCostDetail = new UsageCostDetail();
+    usageCostDetail.setUserId(usageDetail.getUserId());
+    usageCostDetail.setCallCost(usageDetail.getDuration() * this.ratePerSecond);
+    usageCostDetail.setDataCost(usageDetail.getData() * this.ratePerMB);
+    return usageCostDetail;
+    }
+    }
+    ```
+
+#### Testing
+
+You need to set
+
+```
+spring.cloud.stream.bindings.input.destination=test-usage-detail
+spring.cloud.stream.bindings.output.destination=test-usage-cost
+server.port=9091
+```
+
+You can run the standalone `UsageCostProcessor` processor application as,
+
+```
+java -jar usage-cost-processor-0.0.1-SNAPSHOT.jar --spring.cloud.stream.bindings.input.destination=test-usage-detail --spring.cloud.stream.bindings.output.destination=test-usage-cost --server.port=9091
```

Comment:
Following these instructions.  The app starts and then terminates with no exception.  I may have missed something though.